101-simple-windows-vm-monitoring-diagnostics/README.md

```diff
@@ -0,0 +1,21 @@
+# A Windows VM with the Azure diagnostics extension which enables monitoring and diagnostics capabilities 
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-simple-windows-vm-monitoring-diagnostics%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Built by: [sbtron](https://github.com/sbtron)
```

Comment:
This should be removed. This information is captured in the Github history and metadata file